docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1093,90 +907,78 @@ The schema evolution behavior is as follows:
   An error is thrown for incompatible types.
 
 Limitations
-^^^^^^^^^^^
+~~~~~~~~~~~
 
 The following operations are not supported when ``avro_schema_url`` is set:
 
 * ``CREATE TABLE AS`` is not supported.
 * Bucketing(``bucketed_by``) columns are not supported in ``CREATE TABLE``.
 * ``ALTER TABLE`` commands modifying columns are not supported.
 
-.. _hive-procedures:
-
-Procedures
-----------
-
-Use the :doc:`/sql/call` statement to perform data manipulation or
-administrative tasks. Procedures must include a qualified catalog name, if your
-Hive catalog is called ``web``::
-
-    CALL web.system.example_procedure()
-
-The following procedures are available:
-
-* ``system.create_empty_partition(schema_name, table_name, partition_columns, partition_values)``
-
-  Create an empty partition in the specified table.
-
-* ``system.sync_partition_metadata(schema_name, table_name, mode, case_sensitive)``
-
-  Check and update partitions list in metastore. There are three modes available:
+.. _hive-alter-table-execute:
 
-  * ``ADD`` : add any partitions that exist on the file system, but not in the metastore.
-  * ``DROP``: drop any partitions that exist in the metastore, but not on the file system.
-  * ``FULL``: perform both ``ADD`` and ``DROP``.
+ALTER TABLE EXECUTE
+"""""""""""""""""""
 
-  The ``case_sensitive`` argument is optional. The default value is ``true`` for compatibility
-  with Hive's ``MSCK REPAIR TABLE`` behavior, which expects the partition column names in
-  file system paths to use lowercase (e.g. ``col_x=SomeValue``). Partitions on the file system
-  not conforming to this convention are ignored, unless the argument is set to ``false``.
+The connector supports the following commands for use with
+:ref:`ALTER TABLE EXECUTE <alter-table-execute>`.
 
-* ``system.drop_stats(schema_name, table_name, partition_values)``
+**``optimize`` command**
```

Comment:
This renders with the "``" literals instead of like a keyword, at least when I build it on my laptop.